Evmone static library clang build fails while llvm-ar reads mri script

The Solidity fuzzers on oss-fuzz build evmone static libraries (see https://github.com/google/oss-fuzz/blob/8851460f3d7e712a406909acdf8b3aa2882e011d/projects/solidity/Dockerfile#L56-L61) as part of their base docker image. During a [recent update]() from `v0.4.0` -> `0.8.0` (see https://github.com/google/oss-fuzz/pull/6207) it was noticed that evmone-standalone.a build fails (failing log: https://gist.github.com/bshastry/e81e76cc57c393b07accee7599548c88).

The error occurs here

```
2021-08-23T08:54:56.0397220Z [1/2] Linking CXX static library lib/x86_64-linux-gnu/libevmone-standalone.a
2021-08-23T08:54:56.0470727Z FAILED: : && /usr/local/bin/cmake -E rm -f lib/x86_64-linux-gnu/libevmone-standalone.a && /usr/local/bin/llvm-ar qc lib/x86_64-linux-gnu/libevmone-standalone.a   && /usr/local/bin/llvm-ranlib lib/x86_64-linux-gnu/libevmone-standalone.a && cd /src/evmone/build/lib/evmone && /usr/local/bin/llvm-ar -M < evmone-standalone.mri
2021-08-23T08:54:56.0472769Z /usr/local/bin/llvm-ar: error: script line 1: unknown command: OPEN
2021-08-23T08:54:56.0473431Z ninja: build stopped: subcommand failed.
```

Sadly, this is stalling Solidity fuzzing :( 

Pointers to fix the problem welcome :)

--- a/cmake/LibraryTools.cmake
+++ b/cmake/LibraryTools.cmake
@@ -3,7 +3,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
 # For given target of a static library creates a custom target with -standalone suffix
-# that merges the given target and all its static library depenednecies
+# that merges the given target and all its static library dependencies
 # into a single static library.
 #
 # It silently ignores non-static library target and unsupported platforms.
@@ -18,7 +18,7 @@ function(add_standalone_library TARGET)
     if(CMAKE_AR)
         # Generate ar linker script.
         set(script_file ${name}.mri)
-        set(script "OPEN $<TARGET_FILE:${name}>\n")
+        set(script "CREATE $<TARGET_FILE:${name}>\n")
         string(APPEND script "ADDLIB $<TARGET_FILE:${TARGET}>\n")
 
         get_target_property(link_libraries ${TARGET} LINK_LIBRARIES)

